utm_tracker/session.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import Any, List
+
+from django.contrib.sessions.backends.base import SessionBase
+
+from .models import LeadSource
+from .types import UtmParamsDict
+
+SESSION_KEY_UTM_PARAMS = "utm_params"
+
+logger = logging.getLogger(__name__)
+
+
+def stash_utm_params(session: SessionBase, params: UtmParamsDict) -> bool:
+    """
+    Add a new utm_params dictionary to the session.
```

Comment:
This makes it sound like the utm_params key is a dictionary.

"Append a new UtmParamsDict to the session's utm list/stack" ?